src/components/tasks/TaskSearch/Suggestion/Option.tsx

```diff
@@ -0,0 +1,30 @@
+import className from './option.module.scss';
+
+interface IProps {
+    idx: number;
+    onClickHandler: (idx: number) => void;
+    suggestion: { [key: string]: string };
+    activeSuggestionIndex: number;
+}
```

Comment:
Some naming can be improved I think.
`IProps` , `idx`

Any benefit for using `idx` abbreviation instead of `..Index` or `index`?